R/impute.R

```diff
@@ -133,19 +133,19 @@ combine.impute.output = function(inputfile.prefix, outputfile, is.male, imputein
 run_haplotyping = function(chrom, tumourname, normalname, ismale, imputeinfofile, problemloci, impute_exe, min_normal_depth, chrom_names,
                            snp6_reference_info_file=NA, heterozygousFilter=NA) {
   
-  if (file.exists(paste(tumourname, "_alleleFrequencies_chr", chrom, ".txt", sep=""))) {
+  if (file.exists(paste(tumourname, "_alleleFrequencies_", chrom, ".txt", sep=""))) {
```

Comment:
All these naming changes would need to be compared to the battenberg.pl script and modules as they have some pre contig name and post contig name patterns hardcoded in them. Hence me not changing any of the prefixes.

Reply:
that's ok. I worried if they'd break something, hence put all of them in one commit. I'll revert it.